CPractice/Array/array.c

```diff
@@ -0,0 +1,71 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+void init_array(int *base_address, int size);
+void print_array(int *base_address, int size);
+void add_element(int *base_address, int index, int value);
+int get_element(int *base_address, int index);
+void remove_element(int *base_address, int index);
+
+int main(void){
+
+  int size = 5;
```

Comment:
If size won't change, I suggest you use a preprocessor directive:
`#define ARRAY_SIZE     5`
below your `#includes`